### ISSUE TITLE
Error at login

### ISSUE THREAD

**Issue by dgeordgy21** [sunsettrack4/zattoo_tvh#10]
Hello,
using SSH I can't login successfully. Maybe the errors. How should I solve? Tested in Ubuntu 16.04 and 18.04
Thank you

Phantomjs version 2.1.1, installed using this: https://tecadmin.net/install-phantomjs-on-ubuntu/

```
root@vm-sgs:~/ztvh# ./ztvh.sh
ZattooPLUS for tvheadend
(c) 2017-2018 Jan-Luca Neumann
Version 0.4.7 2018/09/11

Checking internet connectivity...
Checking file existence and permissions...

Starting script...

This application failed to start because it could not find or load the Qt platform plugin "offscreen".

Available platform plugins are: phantom.

Reinstalling the application may fix this problem.
PhantomJS has crashed. Please read the bug reporting guide at
<http://phantomjs.org/bug-reporting.html> and file a bug report.
./ztvh.sh: line 397:  1520 Aborted                 (core dumped) phantomjs -platform offscreen ~/ztvh/save_page.js https://zattoo.com/login > cookie_list
Login to Zattoo webservice...

- LOGIN FAILED! PLEASE RETRY! -

- ZATTOO LOGIN PAGE -
email.....:

```

**Comment by sunsettrack4** [comment]
Hello,

you have the following options that will solve the issue:

1) Install fixed v0.4.7+ version:
https://github.com/sunsettrack4/zattoo_tvh/releases/download/v0.4.7/zattoo_tvh-0.4.7+.zip

2) Use the latest version v0.4.8:
https://github.com/sunsettrack4/zattoo_tvh/releases/tag/v0.4.8

Best regards
sunsettrack4

**Comment by dgeordgy21** [comment]
![error](https://user-images.githubusercontent.com/17911815/46651972-fc6d6100-cba1-11e8-8e19-190d0d86adcc.jpg)
Tested with v0.4.8

**Comment by sunsettrack4** [comment]
Could you please try to run the script after removing the following Line?

https://github.com/sunsettrack4/zattoo_tvh/blob/33757de228fe7ad4a783233bb14c763da62094af/ztvh.sh#L248

**Comment by dgeordgy21** [comment]
Still the same error. Cloud you give me a procedure to install PhantomJS? How to (re)install "offscreen" plugin? Or maye a wrong permission on a file? I have already checked. I'm using Zattoo Switzerland.
Thank you!

**Comment by sunsettrack4** [comment]
You could try to export QT_QPA_PLATFORM=phantom in line 248 of ztvh.sh. Please also set -platform phantom in lines 572 and 577.

**Comment by dgeordgy21** [comment]
It seems to work!!! I'm able to watch a TV show, but it apears that some channels are missing.

**Comment by sunsettrack4** [comment]
Thanks for testing! I have to apply this change - please note that I'll release an update today.

**Comment by sunsettrack4** [comment]
Issue fixed in commit ebdaa09481e3b08f0e088085cfa986b9bbd15366
Case closed.
Please re-download latest release.
